src/cljs/commiteth/handlers.cljs

```diff
@@ -62,6 +62,10 @@
  (fn [db [_ path value]]
    (assoc-in db path value)))
 
+(reg-event-db
+ :update-in
```

Comment:
There exists the handler `:assoc-in`, but it should actually, since it breaks the indirection between events and the database. So probably better to create a handler to describe what is intended to. :)